mod/profile.php

```diff
@@ -181,6 +181,10 @@ function profile_content(&$a, $update = 0) {
 		$commpage = (($a->profile['page-flags'] == PAGE_COMMUNITY) ? true : false);
 		$commvisitor = (($commpage && $remote_contact == true) ? true : false);
 
+		if(feature_enabled($a->profile['profile_uid'],'photos_widget')) {
+			require_once('include/photos.php');
```

Comment:
Can you give me some teaching lesson in PHP?
What is the pro and contra to have `require_one` at the different places (e.g. does it have an influence on memory consumption)? Sry for the question but I just want to learn something :-)